pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -113,6 +115,107 @@ type ReconcileNodeNetworkConfigurationPolicy struct {
 	scheme *runtime.Scheme
 }
 
+// It will return the policies with same labels at the one from argument
```

Comment:
You meant to say policies matching the same node/labels? Maybe it would be better to make it policiesMatchingNode.